dbms/tests/integration/test_ttl_replicated/test.py

```diff
@@ -43,17 +44,21 @@ def test_ttl_columns(start_cluster):
     expected = "1\t0\t0\n2\t0\t0\n"
     assert TSV(node1.query("SELECT id, a, b FROM test_ttl ORDER BY id")) == TSV(expected)
     assert TSV(node2.query("SELECT id, a, b FROM test_ttl ORDER BY id")) == TSV(expected)
-    
-def test_ttl_table(start_cluster):
+ 
+@pytest.mark.parametrize("delete_suffix", [
```

Comment:
Seems like we haven't any test for TTL moves?

Reply:
Yes so far.